### PR TITLE
MCE 2.2.10

### PR DESCRIPTION
**Pull request by dockerymick**
Related PR for ACM 2.7.10: https://github.com/stolostron/rhacm-docs/pull/5925/files
Asked release manager about doc issue for this. I need help with confirming which bugs need to be mentioned in the Errata

Query with issues that do not have the `errata` label, but have Errata version in the Fix versions

### CODE REVIEW

**Comment by cadawson** [comment]
It looks like there are no issues to be listed in the errata for MCE this round. I think the one that you pulled in was still open. It will probably have to move ti the next release. I think we go with the standard container updates for this one, I guess. Thanks!

**Review by cadawson — APPROVED** [review]
Looks good. Might be that we only go with the standard updates here and remove the comments. Thanks! I will also add Robyn to our GH group. I don't think she is already there. Thanks!

**Review by oafischer — APPROVED** [review]
Looks good if no issues are intended to be tagged.

**Comment by dockerymick** [comment]
Here is the query: https://issues.redhat.com/browse/ACM-8844?jql=project%20%3D%20ACM%20AND%20fixVersion%20in%20(%22MCE%202.2.10%22)%20

**Comment by cadawson** [comment]
Hey @dockerymick. Just a heads-up that that link is to an issue. It is not the query. This is a link to the query: https://issues.redhat.com/issues/?filter=12425893. I gave you and @rhaignere access to it. Let me know if it doesn't work. Thanks!

**Comment by dockerymick** [comment]
@cadawson the query that you shared is for ACM 2.7.10. 
Screenshot 2023-12-11 at 9 18 34 PM

Why is Ordered by ASC an important label? Does that mean Ascending? 

When I edited the query that you shared to only have MCE, I received no results:
Screenshot 2023-12-11 at 9 22 15 PM

When i removed ACS:

Screenshot 2023-12-11 at 9 23 27 PM

Results after removing errata label:

Screenshot 2023-12-11 at 9 24 30 PM


Results when I added labels ASC, no errata label:

Screenshot 2023-12-11 at 9 25 59 PM

**Comment by cadawson** [comment]
@dockerymick: I got you. The view that it pulled up for your query only showed the single issue. It still does, but I see that I can scroll through the other issues with the arrows in the upper right corner. I did notice the you did not include the errata label. That is how it listed MCE 2.2.10 issues for you. 

Here is the confusion: My query includes both MCE 2.2.10 and ACM 2.7.10, but only the issues that contain the "errata" label. You can separate them into separate MCE and ACM searches, if it helps. Those are the only ones that the docs team is concerned with. Those are the ones that @rhaignere wants us to include because she added the "errata" label to it. In this case, it does not look like there are any MCE 2.2.10 issues for us to include. That might be incorrect until we get the process working, but that fix version and the "errata" label are what we filter on. If it is not there, then we assume that we don't want to expose any of the bug fixes publicly, or there were not any. In that case, we include the standard container updates notification and don't list any bug fixes in the docs errata. 

It might be that you were pulling all of the 2.2.10 issues to help Robyn see which ones were candidates for the errata label. In that case, your query works for what you are looking for. That was cool of you to do that. My query was more selfish in that it is what the doc team needs for the errata.  

You are correct that ASC is ascending order for the issues that match the filters. It should not affect the filtering aside from ordering them in the window. 

Does that help?

**Review by rhaignere — APPROVED** [review]

**Comment by dockerymick** [comment]
@cadawson thanks for reminding me of the process and confirming the query. You are correct about my intentions lol thanks!

File: clusters/release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,14 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. For 2.4, this list includes two ciphers that are _not_ FIPS approved. If you upgrade from a version 2.4.x or earlier and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.
 
+== Errata 2.2.10
+
+* Delivers updates to one or more of the product container images.
+
+//need help listing the bug fixes for MCE 2.2.10
+
+//https://issues.redhat.com/browse/ACM-8844?jql=project%20%3D%20ACM%20AND%20fixVersion%20in%20(%22MCE%202.2.10%22)%20
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]: Removing hidden comment
```suggestion
```

**Comment by dockerymick** [comment]
Merging this PR

**Pull request merged**